UnityProject/Assets/Scripts/Items/Weapons/MagazineBehaviour.cs

```diff
@@ -188,7 +188,7 @@ public virtual void ExpendAmmo(int amount = 1)
 	[Server]
 	public void ServerSetAmmoRemains(int remaining)
 	{
-		SyncServerAmmo(serverAmmoRemains, remaining);
```

Comment:
decently sure this goes against
https://unitystation.github.io/unitystation/development/SyncVar-Best-Practices-for-Easy-Networking/, same goes with all other removals of SyncServerAmmo()

Reply:
thanks ill swap them